### ISSUE TITLE
Doesnt work, Invalid \escape: line 2   column 29 (char 30)

### ISSUE THREAD

**Issue by Cystemiccyanide** [MythicManiac/MythicModManager#10]
https://cdn.discordapp.com/attachments/169228655082471424/571427221420507185/unknown.png

**Comment by Kyusetzu** [comment]
Same here.

![alt text](https://i.imgur.com/USGwHC0.png "")

**Comment by derekhearst** [comment]
Same here

**Comment by derekhearst** [comment]
Found the reason why, need to put backslashes before other ones. 

https://github.com/MythicManiac/MythicModManager/issues/3

**Comment by MythicManiac** [comment]
Yeah this error means the `config.json` file is invalid JSON. You need double `\\` characters, since a singular one is interpreted as an escape prefix
